### ISSUE TITLE
How can we control the latent?

### ISSUE THREAD

**Issue by Jio0728** [CrisHY1995/headnerf#4]
Hello. Thank you for your brilliant model!

As we test the model with our own datasets, we would like to apply your model for more various fitting results.

I have checked that I can rotate the head in circle using the model, and that the result maintains great view consistency.  However, we want to spin the head along the horizontal axis. Is there any way to control the pose or the camera direction in the direction we want?

Also, we cannot use GUI now. We want to control latent codes(expression, identity) and camera direction without using GUI. Could you please give us some advice on what code we should be based on when we construct the relevant code?

**Comment by Jio0728** [comment]
I have an additional question. 

According to your article, it seems like we need to train your model with target's image in order to transform the identity of the source to that of target. I understood that we need to initialize the latent codes with 3DMM result of the target image and train the model to tune the latent codes. 

However, I have no idea how I can extract 4 latent codes using 3DMM for the initialization and how you trained your model to tune the latent codes. Could you please give us some advice on it? 

Thank you.

**Comment by CrisHY1995** [comment]
It is recommended to run the GUI supplied in the code.  Compared to the initial version, this GUI has been improved and contains the functions you need.

Some functions from the "Utils/HeadNeRFUtils.py" file may offer help for controlling latent codes without GUI. 

The code for extracting 4 latent codes using 3DMM has been provided in this repository, and the related commands are shown in the "Data Preprocess" part. Similarly, the code of  fine-tuning latent codes based on Pretrained HeadNeRF has also been provided in the "Fitting a Single Image" part,

![image](https://user-images.githubusercontent.com/49339865/166206558-5495c560-dd3c-46a1-9ce0-f334d171e871.png)

**Comment by Jio0728** [comment]
Thank you for your answers!

Then, does it mean that the result of Fitting3DMM/FittingNL3DMM.py can be used as a target_embedding when fitting a single image?

I want to do linear interpolation with my own target image!
